ElectronClient/gui/NoteContentPropertiesDialog.tsx

```diff
@@ -42,6 +50,8 @@ export default function NoteContentPropertiesDialog(props:NoteContentPropertiesD
 	const [strippedWords, setStrippedWords] = useState<number>(0);
 	const [strippedCharacters, setStrippedCharacters] = useState<number>(0);
 	const [strippedCharactersNoSpace, setStrippedCharactersNoSpace] = useState<number>(0);
+	const [strippedReadTime, setStrippedReadTime] = useState<number>(0);
+	const wordsPerMinute = 200;
```

Comment:
Based on what I've found online 200 wpm is a bit low. Could you change it to 250 please? Also could you add this source as a comment? https://www.researchgate.net/publication/332380784_How_many_words_do_we_read_per_minute_A_review_and_meta-analysis_of_reading_rate